docs/usage.rst

```diff
@@ -2,8 +2,353 @@
 Usage
 =====
 
-To use phenotrex in a project::
+The following quick start example shows training, evaluation and inference with phenotrex models
+using the phenotrex command line interface. For further information on flags and parameters used
+in this tutorial, please consult the relevant CLI documentation available via
+``phenotrex <command> --help``.
 
-    from phenotrex.io import ...  #  file I/O
-    from phenotrex.ml import ...  # classifiers and training/CV functionality
-    from phenotrex.util import ...  # plotting and util functions
+To illustrate phenotrex's capabilities, a small dataset of genome assemblies will be used.
+To download all required data, run:
+
+.. code-block:: console
+
+    $ curl -OL http://fileshare.csb.univie.ac.at/phenotrex/tutorial_data.tar
+    $ tar -xf tutorial_data.tar
+    $ cd tutorial_data
+
+To then install phenotrex, including its capability for extracting features from FASTA files,
+in a new virtual environment:
+
+.. code-block:: console
+
+    $ python3 -m venv env
+    $ source ./env/bin/activate
+    $ pip install phenotrex[fasta]
```

Comment:
For users less familiar with venvs, it might be helpful to give more expressive names (what is the command, what is the name)
```suggestion
.. code-block:: console

    $ python3 -m venv phenotrex-env
    $ source ./phenotrex-env/bin/activate
    $ pip install phenotrex[fasta]
```